javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,12 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  var newArray = [];
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] === 'string') {
+      // do nothing. Is there a better way to do this?
```

Comment:
Yep! Your condition should be `typeof values[i] !== 'string'` so the `if` statement can be where you do the work you want.